### PR TITLE
master-qa-72567

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LRQA-72567
Submit to master.

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagestree/PagesTreeWithPermissions.testcase [hunk 1]
```diff
@@ -172,4 +172,111 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-106110. Pages are not shown when users are not allowed to view certain pages."
+	@priority = "4"
+	test CannotViewPageWhenUserRemovingPermissions {
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagestree/PagesTreeWithPermissions.testcase:177]: @chaowang-123, let's call this `CannotViewChildPageWhenRemovingSiblingPagePermissions`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagestree/PagesTreeWithPermissions.testcase [hunk 2]
```diff
@@ -172,4 +172,111 @@ definition {
...
+			Navigator.openSiteURL(siteName = "Test Site Name");
+
+			PagesFinder.goToPagesFinder();
+
+			for (var i : list "1,2") {
+				PagesFinder.viewPageNameInPagesFinder(pageName = "Test Page Name ${i}");
+			}
+
+			PagesFinder.viewPageNameInPagesFinder(pageName = "Test Child Page 2");
+		}
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagestree/PagesTreeWithPermissions.testcase:279]: At the end, let's also make sure that Test Child Page 1 _doesn't_ display (unless we already have a test that covers child page permissions).

**Review by brianwulbern — COMMENTED** [review]
@chaowang-123 please see my review comments and resend.  Thanks!

**Pull request closed without merging**